<launch>
  <node ns="q1" pkg="multi_uav_exploration" type="multi_uav_exploration_node" name="multi_uav_exploration_node" output="screen">
    <param name="is_relay" type="bool" value="true" />
    <param name="is_sacrifice" type="bool" value="false" />
    <param name="my_ns" type="string" value="/q1/" />
    <param name="other_ns" type="string" value="/q2/" />
  </node>
</launch>
